### ISSUE TITLE
[ELF32]: amacc could not open ELF32 format

### ISSUE THREAD

**Issue by gapry** [jserv/amacc#16]
Although it can compile, it can not do the self-hosting via ELF32 format.

For now, it is the error message

```
[gapry@E130 amacc]$ make check
[ JIT      ]
hello, world
[ compiled ]
could not open(elf/hello)
Makefile:20: recipe for target 'check' failed
make: *** [check] Error 255
```

Is it possible to create a new branch to develop the ELF32 Loader ?

**Comment by jserv** [comment]
It is a known issue reported by @lecopzer .

**Comment by lecopzer** [comment]
Do you mean you can't use elf/amacc which creates by `amacc -o elf/amacc amacc.c` to generate another object file?
Or you only can't do `make check`?

**Comment by gapry** [comment]
For now, I can do `make check` and it can't pass the ptr.c in the testcase. 

```
[*** verify tests/ptr.c <JIT>********]
failed, e - s = 0
passed
[*** verify tests/ptr.c <ELF>********]
[*** verify tests/ptr.c <ELF-self>***]
failed, e - s = 0

```

can you tell me more detail about the isssue which you faced with ?

**Comment by gapry** [comment]
For now, I can do `make check` . ELF and ELF-self can pass the currents test-case.

So, can you tell me more detail about the isssue which you faced with ?

**Comment by gapry** [comment]
fixed via #17
